### PR TITLE
Allow plugins to be updated in case of version = 'latest'

### PR DESCRIPTION
**Pull request by cristifalcas**

### CODE REVIEW

**Comment by mmoll** [comment]
tests fail, please have a look and also rebase onto current master

**Comment by mmoll** [comment]
@cristifalcas what's the status here?

**Comment by cristifalcas** [comment]
Sorry I mist this. I will fix them.

**Comment by mmoll** [comment]
BTW, here's the same situation like in theforeman/puppet-foreman_proxy#218

**Comment by cristifalcas** [comment]
What do you thing it's better? 

To modify foreman::params::version to 'installed' or to update spec tests to check for 'present'?

**Comment by mmoll** [comment]
Personally I have a very slight preference for "present", but I guess if you're asking three people you'll get four answers. :)

**Comment by mmoll** [comment]
@cristifalcas tests still fail. Please also rebase onto current master and squash you commits together, we're getting there :)

File: manifests/plugin.pp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Installs the package for a given Foreman plugin
 define foreman::plugin(
+  $version     = $foreman::version,
```
Comment (ekohl) [inline manifests/plugin.pp]: Should we introduce a ```$foreman::plugin_version``` variable for this?

**Comment by mmoll** [comment]
what's the status here?

**Comment by mmoll** [comment]
ping?

**Comment by cristifalcas** [comment]
i will open a new pr for this issue, as I can't rebase my branch anymore

**Pull request closed without merging**